datus/agent/node/doc_search_node.py

```
@@ -42,11 +68,93 @@ def update_context(self, workflow: Workflow) -> Dict:
+            total_docs = 0
+            for keyword in self.input.keywords:
+                response = self.input.post(url, json=params)
+                response.raise_for_status()
+
+                result = response.json()
+                raw_contents = [result["content"] for result in result.get("results", [])]
+                docs[keyword] = raw_contents
+                total_docs += len(raw_contents)
+
+            return DocSearchResult(success=True, docs=docs, doc_count=total_docs)
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)